azurerm/internal/services/loganalytics/log_analytics_saved_search_resource.go

```diff
@@ -94,6 +94,26 @@ func resourceArmLogAnalyticsSavedSearch() *schema.Resource {
 					),
 				},
 			},
+
+			// tag is defined as an object set and cannot be updated in service side
+			"tag": {
+				Type:     schema.TypeSet,
```

Comment:
since this contains a key and a value we should be able to use the regular `tags.ForceNewSchema()` here instead

ultimately this likely wants to be named `tags` too?